routes/api.php

```diff
@@ -227,8 +227,8 @@
     Route::resource('expenses', ExpenseController::class); // name = (expenses. index / create / show / update / destroy / edit
     Route::put('expenses/{expense}/upload', [ExpenseController::class, 'upload']);
     Route::post('expenses/bulk', [ExpenseController::class, 'bulk'])->name('expenses.bulk');
-
     Route::post('export', [ExportController::class, 'index'])->name('export.index');
+    Route::put('expenses/edocument/upload', [ExpenseController::class, "edocument"])->name("expenses.edocument");
```

Comment:
Hi @turbo124 ,
changed the files according to your help. But still getting 404 not found?

Reply:
I tested this and it works as expected, have you  run ```php artisan optimize``` to update the routes?

Reply:
Yes. Can you send me your curl command to mitigate any incompatibility between python and PHP...?

Reply:
curl -X POST \
 http://ninja.test:8000/api/v1/expenses/edocument/upload \
 -H 'Content-Type: multipart/form-data'\
 -H 'X-API-TOKEN: company-token-test' \
 -H 'X-Requested-With: XMLHttpRequest' \
 -F _method=PUT \
 -F documents[]=@/path/Invoice.xml;